### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by YEEEEEEE777** [shekeru/dotka-cheats#1]
please don't stop

**Comment by lol98xlol98** [comment]
I agree with the above poster Ive been working with this code for about a whole day now straight non stop (for almost 24 hours now)

 ;/ it's very buggy Experienced a lot of crashes when tinkering with the hero and ward pointers.

I attempted an update on a header file as well adding in some not finished functions of which I obtained through other open source projects that had them listed and more well defined.
cbasenpc.h

i'm wondering how would we call vscript variables to be used from classes where we use hero and ward and creep pointers? say if someone wanted to compare hero->m_IVisibleByEnemy
Would we have to write a vscript and add it and return the value to the function using the hero pointer? I really wanted to stay away from vscripts but if it is what must be done to access scripting variables I guess I will have to implement that.

This is the base of a really fucking nifty tool though. I was almost considering going over to linux prior to finding this.

I cant seem to get wards to show any kind of debugoverlays whether they are visible to me or not.
Im wondering if i need to create a vector and somehow get their location then destroy their model and reload them with additional flags or something.

Right now Im working on trying to get illusions to always show, I wanted to scale them down highly or place the original "on fire" or something, I will figure it out eventually. as the boxes are quite a fuckin mess and dont render half of the time for me using opengl- however it runs the best for me as I have a shit computer.

As far as invisible flags for hero is there a way to show enemy is somewhere nearby even though it is invisible? I do not notice the client destroying the hero as if it were too far out of screen or dead, which makes me believe we can force some type of graphic overlay onto an invisible hero as well.

I am also very thankful that this tool is nowhere near a final stage, gotta make a mf work for it. :)
I am willing to colab with anyone who wants to work on this shit I've been into the gamehacking scene for quite some time, been away from a compiler up until yesterday for a couple years though. It's almost like riding a bike you don't really forget.

**Comment by snajdovski** [comment]
Does anyone want to work with me together on this, I have created Javascript API vulnerability script and inject some scripts check out on my github page free feel to fork it and use it and improve it, I have made like 30+ scripts.
We can combine these and create something cool

**Comment by Wolf49406** [comment]
Thanks!
![image](https://user-images.githubusercontent.com/26302888/209752646-368106f1-4521-4471-a19a-43b824f8a560.png)
